src/huggingface_hub/hf_api.py

```diff
@@ -5357,6 +5391,21 @@ def duplicate_space(
                 Hugging Face token. Will default to the locally saved token if not provided.
             exist_ok (`bool`, *optional*, defaults to `False`):
                 If `True`, do not raise an error if repo already exists.
+            hardware (`SpaceHardware` or `str`, *optional*):
+                Choice of Hardware. Example: `"t4-medium"`. See [`SpaceHardware`] for a complete list.
+            storage (`SpaceStorage` or `str`, *optional*):
+                Choice of persistent storage tier. Example: `"small"`. See [`SpaceStorage`] for a complete list.
+            sleepTimeSeconds (`int`, *optional*):
```

Comment:
```suggestion
            sleep_time (`int`, *optional*):
```

Same here, `sleep_time` would be best :)

Reply:
Changed this to `sleep_time` throughout. Also updated the test case.